package.json

```diff
@@ -101,5 +111,9 @@
   "resolutions": {
     "json5": "^2.2.2",
     "optionator": "^0.9.3"
+  },
+  "lint-staged": {
+    "*.{js,ts}": "npm run lint",
+    "*.{ts,tsx,js,css,md}": "prettier --write"
```

Comment:
Do we need to be concerned abt `tsx` files in this repo?

Reply:
Nope, can remove was a copy mistake